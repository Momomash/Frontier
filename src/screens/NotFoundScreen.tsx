import React, { FunctionComponent } from 'react';

type Props = {};
type State = {};

export const NotFoundScreen: FunctionComponent<Props> = () => {
    return <p>404 !</p>;
};
