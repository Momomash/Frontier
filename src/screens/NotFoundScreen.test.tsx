import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { NotFoundScreen } from './NotFoundScreen';

describe('NotFoundScreen', () => {
    it('render NotFoundScreen', () => {
        expect(renderer.create(<NotFoundScreen />).toJSON()).toMatchSnapshot();
    });
});
