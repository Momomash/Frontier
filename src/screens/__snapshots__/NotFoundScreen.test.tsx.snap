// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundScreen render NotFoundScreen 1`] = `
<p>
  404 !
</p>
`;
